sql/core/src/test/resources/sql-tests/inputs/group-analytics.sql

```diff
@@ -18,9 +18,11 @@ AS courseSales(course, year, earnings);
 
 -- ROLLUP
 SELECT course, year, SUM(earnings) FROM courseSales GROUP BY ROLLUP(course, year) ORDER BY course, year;
+SELECT course, year, SUM(earnings) FROM courseSales GROUP BY ROLLUP (course, year) ORDER BY course, year;
```

Comment:
We need end-2-end tests for this? Instead, how about moving this kind of tests into `PlanParserSuite`?

Reply:
+1, I don't think we should test extra space in the end-to-end test.

Reply:
> +1, I don't think we should test extra space in the end-to-end test.

Done